qiskit_experiments/calibration/experiments/rough_amplitude.py

```diff
@@ -0,0 +1,130 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Rough amplitude calibration."""
+
+from dataclasses import asdict
+from datetime import datetime
+from typing import Dict, List, Optional
+import numpy as np
+
+from qiskit.pulse import DriveChannel
+from qiskit import QuantumCircuit
+from qiskit_experiments.base_experiment import BaseExperiment
+from qiskit_experiments.calibration.analysis import CosineFit
+from qiskit_experiments.calibration.analysis.utils import get_period_fraction
+from qiskit_experiments.calibration.metadata import CalibrationMetadata
+from qiskit_experiments.calibration.calibration_definitions import CalibrationsDefinition
+from qiskit_experiments.calibration.data_processing.data_processor import DataProcessor
+from qiskit_experiments.calibration.parameter_value import ParameterValue
+from qiskit_experiments import ExperimentData
+
+
+class RoughAmplitude(BaseExperiment):
+    """Rough amplitude calibration that scans the amplitude."""
+
+    # Analysis class for experiment
+    __analysis_class__ = CosineFit
+
+    def __init__(self,
+                 qubit: int,
+                 cals: CalibrationsDefinition,
+                 gate_name: str,
```

Comment:
This is a good idea but needs to be extended. In your example we also need to specify the angle for the rabi calibration. I suggest adding an extra field called `options` so that your example above becomes:
```
__calibration_objective__ = {
    'gates': ['x90p', 'xp'],
    'options': [np.pi/2, np.pi],
    'parameter_name: 'amp'
}
```

Reply:
See c67acbd

Reply:
On second thoughts I'm not sure a class variable is the way to go here. Let's discuss offline.

Reply:
See e.g. fa3f3eb for a follow up.

Reply:
Let's go for an instance variable.